src/components/Button/src/Button.vue

```diff
@@ -51,38 +51,77 @@ function getContrast(chromaBg, targetChromaFg) {
 }
 
 function getFocus(chromaColor) {
-	return chromaColor.alpha(0.8);
+	return chromaColor.alpha(0.3);
+}
+
+function getHover(chromaColor) {
+	// mix color with 5% black
+	return chroma.mix(chromaColor, '#000', 0.05);
+}
+
+function getActive(chromaColor) {
+	// mix color with 10% black
+	return chroma.mix(chromaColor, '#000', 0.1);
 }
 
 function fill(tokens) {
 	const color = chroma(tokens.color);
+	const colorHover = getHover(color);
+	const colorActive = getActive(color);
 	const textColor = tokens.textColor ? chroma(tokens.textColor) : undefined;
 	const contrastColor = getContrast(color, textColor);
```

Comment:
Didn't know we were doing basic contrast checking in Maker. This is better than nothing, thought it doesn't line up with how the contrast works in website-springboard as there is more nuance to the color flipping.

Reply:
yep, we will probably import the contrast checks from website to maker or from maker to website at some point.